### ISSUE TITLE
Tb78: current mail account is not preselected

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/SmartTemplate4#112]
In Thunderbird 68, when opening SmartTemplates settings, the current identity / (default identity of current folder's account) is preselected. This will also expand the dropdown menu horizontally as the dropdown does auto-resize. See also issue #110 on a similar topic.

Definitely the Thunderbird 78 version of SmartTemplates should preselect the current account identity too.

**Comment by RealRaven2000** [comment]
Preview version with fix:

[smartTemplate-fx-3.4pre5.zip](https://github.com/RealRaven2000/SmartTemplate4/files/5765053/smartTemplate-fx-3.4pre5.zip)

**Comment by macdeport** [comment]
https://github.com/RealRaven2000/SmartTemplate4/issues/110#issuecomment-754092796

**Comment by RealRaven2000** [comment]
I am going to sneak this into the current release as 3.3.1 - since 3.3 wasn't reviewed yet this won't create extra work for the reviewers. Note: you won't be auto-updated when it drops as you have 3.4 installed (higher version number). So check near the end of the week and force install over or reinstall 3.2 to await the automatic update.

i got some bigger fish to fry for 3.4 but having a clear account selector is very important for new users.

**Comment by RealRaven2000** [comment]
implemented in 3.3.1 - released and reviewed on atn today

**Comment by macdeport** [comment]
https://github.com/RealRaven2000/SmartTemplate4/issues/112#issuecomment-754214904
But still opening systematically in **Common** folder with 3.4pre30 version (if I am not making some mistake).

**Comment by RealRaven2000** [comment]
> [#112 (comment)](https://github.com/RealRaven2000/SmartTemplate4/issues/112#issuecomment-754214904)
> But still opening systematically in **Common** folder with 3.4pre30 version (if I am not making some mistake).

It depends what mail folder you are in when you open settings? The current mail folder determines which account you are currently in. If you have activated "use common" in that account, then the common settings are opened instead. The idea is to open the settings that apply if you would reply to an email in that folder / create a new email from the folder.

**Comment by macdeport** [comment]
> It depends what mail folder you are in when you open settings? The current mail folder determines which account you are currently in. If you have activated "use common" in that account, then the common settings are opened instead. The idea is to open the settings that apply if you would reply to an email in that folder / create a new email from the folder.

Of course, good to recall this, but the point is another folder is activated in that folder and it is why I tell to you ;-)

**Comment by RealRaven2000** [comment]
> > It depends what mail folder you are in when you open settings? The current mail folder determines which account you are currently in. If you have activated "use common" in that account, then the common settings are opened instead. The idea is to open the settings that apply if you would reply to an email in that folder / create a new email from the folder.
> 
> Of course, good to recall this, but the point is _another account is activated in that folder_ and it is why I tell to you ;-)

I don't understand? what does " _another account is activated in that folder_ " mean? Where do you open settings from? I assume from the main window (not from the composer window). Every folder has parents, the account is at the top. Are you talking about local folders?

**Comment by macdeport** [comment]
> Are you talking about local folders?
Yes

Sorry, I am not clear because I need  to elaborate as I try (I "dream)) to use in synergy between _SmartTemplate⁴_ & _QuickFolders_ and in fact it is a big feature request...

So I am in local folders _"Mac OS X"_ window:
![thunderbird78-macosx-window](https://user-images.githubusercontent.com/26068595/107068427-e4916080-67e0-11eb-9a48-b73ba2b60a28.jpg)

Then I open SmartTemplate⁴ setting:
![smarttemplate-3 4pre30-setting](https://user-images.githubusercontent.com/26068595/107069581-7352ad00-67e2-11eb-9009-bef826eecba5.jpg)

My hope is to open quite logically with the account I have selected in [Advanced Properties] setting  for this peculiar folder...

**Comment by RealRaven2000** [comment]
Well, sorry that's impossible. Neither Thunderbird nor my Add-ons know that your local folder named "Mac OS X" belongs to a "real" mail account. How would it know this?

**Comment by RealRaven2000** [comment]
There is a problem with the preselected Account: the [x] use common checkbox is disabled - it shouldn't be.

I raised #121 to handle it.

**Comment by macdeport** [comment]
> Well, sorry that's impossible. Neither Thunderbird nor my Add-ons know that your local folder named "Mac OS X" belongs to a "real" mail account. How would it know this?
> 
> In Local folders I always choose the "Common" settings.
> 

I raised ??? to answer to you as it is a feature request
